src/App.tsx

```diff
@@ -1,79 +1,53 @@
-// Copyright 2022 Cartesi Pte. Ltd.
-
-// Licensed under the Apache License, Version 2.0 (the "License"); you may not
-// use this file except in compliance with the License. You may obtain a copy
-// of the license at http://www.apache.org/licenses/LICENSE-2.0
-
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-// License for the specific language governing permissions and limitations
-// under the License.
-
-import { FC, useEffect } from "react";
-import injectedModule from "@web3-onboard/injected-wallets";
-import { init, useSetChain } from "@web3-onboard/react";
+import { FC } from "react";
 import { useState } from "react";
 
-import { GraphQLProvider } from "./GraphQL";
-import { Notices } from "./Notices";
-import { Input } from "./Input";
-import { Inspect } from "./Inspect";
 import { Network } from "./Network";
-import { Vouchers } from "./Vouchers";
+// import { Account, WalletOptions } from "./Network";
+import { Inspect } from "./Inspect";
+import { Input } from "./Input";
+import { Portals } from "./Portals";
 import { Reports } from "./Reports";
-import configFile from "./config.json";
+import { Notices } from "./Notices";
+import { Vouchers } from "./Vouchers";
 
-const config: any = configFile;
+const App: FC = () => {
+    // const [{ connectedChain }] = useSetChain();
+    const [appAddress, setAppAddress] = useState<`0x${string}`|undefined>("0xab7528bb862fb57e8a2bcd567a2e929a0be56a5e");
```

Comment:
I have mixed feelings about this address hardcoded

Reply:
Well, it facilitates local development
Otherwise every time we reload the page we would have to edit it
If you really don't want that, we can remove it